### ISSUE TITLE
Do `ClassName`, `AttrName`, and `PropName` types belong here instead of `web-html`?

### ISSUE THREAD

**Issue by nsaunders** [purescript-web/purescript-web-dom#56]
purescript-web/purescript-web-html#30 migrated the `ClassName`, `AttrName`, and `PropName` types from [Halogen](https://github.com/purescript-halogen/purescript-halogen) to [web-html](https://github.com/purescript-web/purescript-web-html).

Considering several relevant functions in `Web.DOM.Element` ([`className`](https://pursuit.purescript.org/packages/purescript-web-dom/6.0.0/docs/Web.DOM.Element#v:className), [`setClassName`](https://pursuit.purescript.org/packages/purescript-web-dom/6.0.0/docs/Web.DOM.Element#v:setClassName), [`getAttribute`](https://pursuit.purescript.org/packages/purescript-web-dom/6.0.0/docs/Web.DOM.Element#v:getAttribute), [`setAttribute`](https://pursuit.purescript.org/packages/purescript-web-dom/6.0.0/docs/Web.DOM.Element#v:setAttribute), [`hasAttribute`](https://pursuit.purescript.org/packages/purescript-web-dom/6.0.0/docs/Web.DOM.Element#v:hasAttribute), and [`removeAttribute`](https://pursuit.purescript.org/packages/purescript-web-dom/6.0.0/docs/Web.DOM.Element#v:removeAttribute)), I wonder why web-html was determined to be the right place for these types rather than web-dom?

And, for that matter, should the above-listed functions not make use of these `ClassName` and `AttrName` types? (The situation today is understandable, of course, in that web-dom can't/shouldn't depend on web-html.)

**Comment by garyb** [comment]
Yeah, moving those here makes sense to me. It could be that they were created on the other repo because Miles was unaware/forgot about the existence of this one :smile:
